src/core/settings.py

```diff
@@ -126,6 +133,10 @@ def model_post_init(self, __context: Any) -> None:
                     if self.DEFAULT_MODEL is None:
                         self.DEFAULT_MODEL = OpenAIModelName.GPT_4O_MINI
                     self.AVAILABLE_MODELS.update(set(OpenAIModelName))
+                case Provider.OPENAI_COMPATIBLE:
+                    if self.DEFAULT_MODEL is None:
+                        self.DEFAULT_MODEL = OpenAICompatibleName.OPENAI_COMPATIBLE
+                    self.AVAILABLE_MODELS.update(set(OpenAICompatibleName))
```

Comment:
Should we add some lightweight validation checks here that the needed settings were set?

Reply:
You are right, it has been added